erpc_c/transports/erpc_rpmsg_lite_base_transport.h

```diff
@@ -29,6 +30,9 @@
 // Classes
 ////////////////////////////////////////////////////////////////////////////////
 
+/* TODO: replace bellow size representing sizeof(rpmsg_queue_rx_cb_data_t)*/
+#define SIZE_OF_RPMSG_QUEUE 12
```

Comment:
@MichalPrincNXP we need get rid of this macro. Any idea?

Reply:
There could be 
#define RL_ENV_QUEUE_STATIC_STORAGE_SIZE (RL_BUFFER_COUNT * sizeof(rpmsg_queue_rx_cb_data_t))
placed in rpmsg_env_specific.h 

and then we do not need the SIZE_OF_RPMSG_QUEUE macro and instead we could use 
uint8_t m_queue_stack[RL_ENV_QUEUE_STATIC_STORAGE_SIZE];
in RPMsgBaseTransport class

I am just testing the static allocation update in rpmsg_lite and once it is stable we can focus on erpc adaptation.

Reply:
SIZE_OF_RPMSG_QUEUE can be removed